lib/svc/method/net-physical

```diff
@@ -530,6 +533,15 @@ if smf_is_globalzone; then
     # Make any mtu adjustments that may be necessary
     setup_mtu
 
+    # Create overlay_rules.json
+    config_overlays="$(declare -p | grep "\ CONFIG_overlay_[a-zA-Z0-9_]*=" | cut -d "_" -f 2- | cut -d "=" -f 1)"
```

Comment:
Please change the regex to the one present in nictagadm: `'^[a-zA-Z0-9_]*[a-zA-Z_]$'`. We don't want to create invalid overlay_rules.json.
We should probably issue a warning if invalid overlay name is specified (to better debug problem with overlay rule not created).

Reply:
The regexp makes sense.
In case of a problem one can check /var/svc/log/network-physical:default.log.